nodes/core/core/75-exec.js

```diff
@@ -130,7 +130,8 @@ module.exports = function(RED) {
                         if (isUtf8(msg.payload)) { msg.payload = msg.payload.toString(); }
                         var msg2 = null;
                         if (stderr) {
-                            msg2 = {payload: stderr};
+                            msg2 = RED.util.cloneMessage(msg);
```

Comment:
if `msg.payload` is big, this clone will needlessly copy it, only for it to be discarded in the next line.

I suggest moving the creation of `msg2` up a bit:

```
child = exec(cl, {encoding: 'binary', maxBuffer:10000000}, function (error, stdout, stderr) {
    var msg2;
    delete msg.payload; // get rid of the old payload we're about to overwrite
    if (stderr) {
        // clone the minimal msg object
        msg2 = RED.util.cloneMessage(msg);
        msg2.payload = stderr;
    }
    msg.payload = Buffer.from(stdout,"binary");
    ...
```

Reply:
Changed as suggested